src/calculateRentalCost.js

```diff
@@ -1,10 +1,13 @@
-/**
- * @param {number} days
- *
- * @return {number}
- */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= 50;
```

Comment:
Use constants for the number of days and the discount amount to avoid hardcoding these values. This will make the code more readable and easier to maintain.